src/ink.tsx

```diff
@@ -53,19 +54,13 @@ export default class Ink {
 
 		this.rootNode.onRender = options.debug
 			? this.onRender
-			: throttle(this.onRender, 32, {
-					leading: true,
-					trailing: true,
-				});
+			: throttle(this.onRender, 32);
 
 		this.rootNode.onImmediateRender = this.onRender;
 		this.log = logUpdate.create(options.stdout);
 		this.throttledLog = options.debug
 			? this.log
-			: throttle(this.log, undefined, {
-					leading: true,
-					trailing: true,
-				});
+			: throttle(this.log, undefined);
```

Comment:
I would just do this to make it simple:

```suggestion
			: (throttle(this.log, undefined) as LogUpdate);
```